packages/accordion/README.md

```diff
@@ -0,0 +1,66 @@
+# @littlespoon/accordion
+
+[![NPM version](https://img.shields.io/npm/v/@littlespoon/accordion.svg)](https://www.npmjs.com/package/@littlespoon/accordion)
+
+## Install
+
+[Yarn](https://yarnpkg.com/package/@littlespoon/accordion):
+
+```sh
+yarn add @littlespoon/accordion
+```
+
+[npm](https://www.npmjs.com/package/@littlespoon/accordion):
+
+```sh
+npm install @littlespoon/accordion --save
+```
+
+[UNPKG](https://unpkg.com/browse/@littlespoon/accordion/)
+
+## Usage
+
+```ts
+import Accordion from '@littlespoon/accordion'
```

Comment:
Can you give an example on how to import `AccordionItem`, `AccordionSummary`, and `AccordionDetails`?